addons/mrp/report/mrp_report_mo_overview.py

```diff
@@ -340,7 +340,8 @@ def _add_transit_line(self, move_raw, forecast, production, level, current_index
 
         product = move_raw.product_id
         currency = (production.company_id or self.env.company).currency_id
-        receipt_date = datetime.strptime(in_transit['delivery_date'], '%m/%d/%Y')
+        lg = self.env['res.lang']._lang_get(self.env.user.lang) or get_lang(self.env)
```

Comment:
I was wondering why it's not just `get_lang`, but then I found similar example in `web_editor`

https://github.com/odoo/odoo/blob/84ac1d1093523a902be7de1c3522adcc01ebb6f8/addons/web_editor/models/ir_qweb_fields.py#L279